src/index.js

```diff
@@ -0,0 +1,161 @@
+          <span>${item.creation_date} </span>
+          <span class="user-name">${item.username}: </span>
+          <span>${item.comment}</span>
+        </div>
+      `;
+      return commentList.append(itemList);
+    });
+  }
+};
+
+const getTotal = async () => {
+  const { data, items, search } = await totalBooksItems();
+  return { data, items, search };
+};
+
+const bookDetails = async () => {
+  const { data, items, search } = await getTotal();
+
+  const totalBooks = document.querySelector('.total-books');
+
+  totalBooks.innerHTML = `(${items.length}) Books about ${search}`;
+
+  for (let i = 0; i < data.items.length; i += 1) {
+    const likes = likesData.filter((item) => item.item_id === data.items[i].id);
+
+    const bookList = document.getElementById('book-list');
+    const bookCard = document.createElement('div');
+    const contentCard = document.createElement('p');
+
+    const title = document.createElement('h4');
+    const bookCardContent = `
+            <p>
+                <a  href='${data.items[i].volumeInfo.previewLink}'>
+                    <img src='${data.items[i].volumeInfo.imageLinks.thumbnail}' class="book-cover">
+                    <br>More details
+                </a>
+            </p>
+            <button type="submit" class="add-comment" id="comment-btn-${data.items[i].id}">Comment</button>
+            &#160;&#160;&#160;&#160;
+            <button class="heart" id="heart-${data.items[i].id}">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="white" class="bi bi-heart-fill" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M8 1.314C12.438-3.248 23.534 4.735 8 15-7.534 4.736 3.562-3.248 8 1.314z"/>
+                </svg>
+            </button>
+            &#160;&#160;&#160;&#160;
+            <span class="likes-counter">${likes.length === 0 ? 0 : likes[0].likes}</span>
+            &#160;&#160;&#160;
+            <span class="likes">likes</span>`;
+    const title1 = `${data.items[i].volumeInfo.title}`;
+    title.classList.add('title');
+    title.innerHTML = title1;
+    bookCard.appendChild(title);
+    bookCard.classList.add('bookCard');
+    contentCard.innerHTML = bookCardContent;
+    bookCard.appendChild(contentCard);
+    bookList.appendChild(bookCard);
+    const commentBtn = document.getElementById(`comment-btn-${data.items[i].id}`);
+    commentBtn.addEventListener('click', () => {
+      popUp(data.items[i]);
+    });
+
+    const heartBtn = document.getElementById(`heart-${data.items[i].id}`);
+    heartBtn.addEventListener('click', () => {
+      createLike(data.items[i].id);
+    });
+  }
+};
+
+const retrieveBooks = () => {
+  bookList.innerHTML = '';
+  bookDetails();
+};
+
+const searchInputBtn = document.getElementById('btn-search');
+searchInputBtn.addEventListener('click', retrieveBooks);
+
+function loadLogo() {
+  const googleLogo = document.querySelector('.header-img');
+  googleLogo.src = Logo;
+  googleLogo.alt = 'google books Logo';
+}
```

Comment:
- [ ] Please, make sure you use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)